schemachange/cli.py

```diff
@@ -199,7 +199,7 @@ def get_all_scripts_recursively(root_directory, verbose):
       if script_type == 'R':
         script['script_description'] = repeatable_script_name_parts.group(2).replace('_', ' ').capitalize()
       elif script_type == 'D':
-        script['script_description'] = delta_script_name_parts.group(2).replace('_', ' ').capitalize()
+        script['script_description'] = delta_script_name_parts.group(3).replace('_', ' ').capitalize()
       else:
         script_name_parts.group(3).replace('_', ' ').capitalize()
```

Comment:
i think the bug was on this line

Reply:
Actually it was correct. 
For d and r it should be group 2 and for v group 3. 
As with v group 2 is the version.

Reply:
@sabumaru is right. What i meant was that else is wrong...
`else:
        script_name_parts.group(3).replace('_', ' ').capitalize()`

should be
`else:
        script['script_description'] = script_name_parts.group(3).replace('_', ' ').capitalize()`

Reply:
sry I am blind